frontendservice/src/views/CustomerHome.vue

```diff
@@ -7,48 +7,168 @@ CONDITIONS OF ANY KIND, either express or implied. See the License for the
 specific language governing permissions and limitations under the License. */
 
 <template>
-  <div class="home">
-    <h1>Welcome customer!</h1>
-    <p>
-      This page will be Home page for the customers. 
-    </p>
-    <p>
-      ToDo: Implement CustomerHome page features including NavBar, Search Bar, MyTokens and AllShops.
-    </p>  
-    <div>
-      <router-link to="/my-tokens">My Tokens</router-link>
+  <div v-if="!isLoggedIn">
+    <NavBarCustomer></NavBarCustomer>
+    <cookieinfo></cookieinfo>
+    <cookieackno></cookieackno>
+    <div id="searchBarWrap" style="margin: 20px 0">
+      <div class="field has-addons">
+        <div class="form-group has-feedback">
+          <i class="fa fa-search" style="font-size:32px"></i>
+          <input id="searchBar" class="input" type="text" placeholder="Find a shop" />
```

Comment:
Nit. For unification purposes: Ishita has written "search shops" in placeholder and she does not have icon here. Please talk to her and make the search bars similar.

Reply:
okay